src/components/featureHighlight/index.js

```diff
@@ -136,56 +145,82 @@ class FeatureHighlight extends BaseComponent {
     return findNodeHandle(target);
   }
 
+  animate(toValue) {
+    Animated.timing(                  // Animate over time
+      this.state.fadeAnim,            // The animated value to drive
+      {
+        toValue,                      // Animate to value
+        duration: toValue === 0 ? 0 : 100,   // Make it take a while
```

Comment:
toValue is either 1 or 0, it already acts as a boolean. We can omit the `=== 0` (and change the order of condition)